docs/index.rst

```diff
@@ -69,6 +69,35 @@ Example:
    to this mount.
 
 
+CDN
+~~~
+
+Crane can also be configured to support some advanced options when content resides
+on a Content Delivery Network. The URL match and replace parameters support rewriting
+the base paths of repos to a new location on the CDN.
+
+The URL authorization options allow Crane to append a query string authorization parameter
+based on a shared secret in order to allow content to be restricted on the CDN.
+
+The following options should go under a section named ``[cdn]``
+
+url_match
+  optional portion of URL to match when rewriting repos' base URLs
+
+url_replace
+  optional replacement for ``url_match`` when rewriting the repos' base URLs. This will
+  only be used when specified along with ``url_match``
+
+url_auth_secret
+  seed used for generating HMAC tokens for URL authorization on CDN. This follows the
+  format used by Akamai's Token Authorization 2.0 feature
+
+url_auth_param
```

Comment:
is this also part of Akamai's Token Authorization?
what's the purpose of this param?